src/messaging/ExchangeContext.cpp

```diff
@@ -573,5 +579,19 @@ ExchangeMessageDispatch & ExchangeContext::GetMessageDispatch(bool isEphemeralEx
     return ApplicationExchangeDispatch::Instance();
 }
 
+void ExchangeContext::AbortAllOtherCommunicationOnFabric()
+{
+    if (!mSession || !mSession->IsSecureSession())
+    {
+        return;
+    }
```

Comment:
```suggestion
    VerifyOrDie(mSession && mSession->IsSecureSession());
```

Can it die here ?

If this check fails, it probably doesn't work as caller expects.

Reply:
I agree it probably does not work as caller expects, but I am also a little leery of dying in public APIs.  We _could_ add error logging, though; might be a good idea.